pkg/profilecreator/profilecreator.go

```diff
@@ -189,3 +195,21 @@ func GetMCP(mustGatherDirPath, mcpName string) (*machineconfigv1.MachineConfigPo
 	}
 	return &mcp, nil
 }
+
+// LoadSnapshot loads a ghw snapshot corresponding to a node
+func LoadSnapshot(mustGatherDirPath string, node *v1.Node) (*option.Option, error) {
```

Comment:
so this doesn't actually load a snapshot, but rather creates the options needed to consume it

Reply:
Ack!